### DISCUSSION TITLE
Can I get rid of author bio in main page?

### DISCUSSION THREAD

**Discussion by inmare** [mmistakes/minimal-mistakes#2659] [Q&A] [answered]
I want to remove author bio in main page (first picture) and replace it to sidebar menu like second picture.
I found similar issue in #6, but it's too old and does not match with newest version.
How could I remove and replace it? Also, thanks for good theme.
![image](https://user-images.githubusercontent.com/62546714/90332162-b8491400-dff5-11ea-8760-a99531be0266.png)
![image](https://user-images.githubusercontent.com/62546714/90332168-c139e580-dff5-11ea-995d-236ea346dfbe.png)

**Comment by mmistakes** [comment]
The documentation explains how:

https://mmistakes.github.io/minimal-mistakes/docs/layouts/#custom-sidebar-navigation-menu

And here is a sample post with the YAML needed to replicate: 

https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_posts/2012-03-15-layout-sidebar-nav-list.md

**Comment by inmare** [comment]
Thanks. But I also tried it already, and it didn't remove author bio and replace it to sidebar menu. 
This is my site and repos.
https://yatsumehole.github.io/Aviutl-info/
https://github.com/yatsumehole/Aviutl-info

**Comment by iBug** [comment, marked as answer]
For the home page, you need to clean up the file bundled in the repository:

<https://github.com/yatsumehole/Aviutl-info/blob/gh-pages/index.html>

**Comment by inmare** [comment]
Thank you so much! It works!
